site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties

```diff
@@ -1029,3 +1030,49 @@ gen.listnavselect = To operate the combo box, first press Alt+Down Arrow to open
 search.label=Search
 search.button=Search
 search.clear=Clear Search
+
+help-tips-header=While you wait, enjoy some cheer from the Isidore Cheer Cup!
+help-tips-favorite-header=Favoriting Isidore Course Sites
+help-tips-favorite=Click on the star next to course sites you want to add to your favorites. Favoriting these sites will pin them to the top blue menu bar in Isidore.
```

Comment:
What does the help tips for favouriting sites have to do with the Overview page? Why is it included in this PR?